### PR TITLE
BREW-691 / Kobold Press; Warlock 02 Dread Magic

### PR DESCRIPTION
**Pull request by Spappz**
Lyra can have his KPSC semi-required expansion now 😊

~~don't tell him that _this_ is also dependent on another unconverted KP source 👀💦~~

### CODE REVIEW

**Comment by revilowaldow** [comment]
> Lyra can have his KPSC semi-required expansion now 😊

You missed one
image

File: collection/Kobold Press; Warlock 02 Dread Magic.json [hunk 1]
```diff
@@ -0,0 +1,2146 @@
...
+			"charges": 7,
+			"recharge": "dawn",
+			"rechargeAmount": "1d4 + 3",
+			"fluff": {
+				"images": [
+					{
+						"type": "image",
+						"href": {
+							"type": "external",
+							"url": "https://raw.githubusercontent.com/Spappz/homebrew/w02dm/_img/W02DM/item/taergashs tear.webp"
```
Comment (revilowaldow) [inline collection/Kobold Press; Warlock 02 Dread Magic.json:158]:
```suggestion
							"url": "https://raw.githubusercontent.com/TheGiddyLimit/homebrew/master/_img/W02DM/item/taergashs tear.webp"
```
Throughout

**Comment by revilowaldow** [comment]
Please could you also fix the `#TODO` on line 10321 of KPSC to reference/loader tag this new source. Thanks
> `some believe Arshin the Enchanter\u2014see {@i Warlock #2} {@note #TODO BREW-691}\u2014`

File: collection/Kobold Press; Warlock 02 Dread Magic.json [hunk 2]
```diff
@@ -0,0 +1,2146 @@
...
+							"entries": [
+								"The strange square pillars of the City of Blue Blocks are 400 feet wide and 200 feet high and are made from a translucent blue stone not seen anywhere else in Midgard. The columns are without seams or joins and have proved impervious to damage.",
+								"Ever since the pillars appeared on the island in the middle of Lake Leukos, they have been a source of great interest to the wizards of Bemmea. Expeditions were dispatched from the Academies Arcana to study the stone blocks, and a small community and trading hub has grown up around the pillars, which continue to fascinate arcanists to this day.",
+								"Recently, rumors that several of the infamous Halaaz Tablets are perhaps hidden here have drawn adventurers and would-be treasure hunters to the city. The most notable features of the city are described below:",
+								{
+									"type": "image",
+									"href": {
+										"type": "external",
+										"url": "https://raw.githubusercontent.com/Spappz/homebrew/w02dm/_img/W02DM/place/blue block city.webp"
+									}
```
Comment (revilowaldow) [inline collection/Kobold Press; Warlock 02 Dread Magic.json:1974]:
```suggestion
									},
									"mapRegions": [
										{
											"area": "W02DM2",
											"points": [
												[
													475,
													580
												],
												[
													588,
													580
												],
												[
													638,
													594
												],
												[
													654,
													660
												],
												[
													667,
													660
												],
												[
													667,
													712
												],
												[
													645,
													734
												],
												[
													607,
													734
												],
												[
													607,
													767
												],
												[
													551,
													767
												],
												[
													551,
													738
												],
												[
													526,
													713
												],
												[
													459,
													713
												],
												[
													459,
													580
												]
											]
										},
										{
											"area": "W02DM3",
											"points": [
												[
													260,
													228
												],
												[
													260,
													310
												],
												[
													196,
													310
												],
												[
													196,
													450
												],
												[
													172,
													450
												],
												[
													172,
													575
												],
												[
													288,
													575
												],
												[
													288,
													517
												],
												[
													441,
													517
												],
												[
													441,
													456
												],
												[
													591,
													456
												],
												[
													591,
													350
												],
												[
													504,
													350
												],
												[
													504,
													280
												],
												[
													382,
													280
												],
												[
													382,
													228
												]
											]
										},
										{
											"area": "W02DM1",
											"points": [
												[
													517,
													460
												],
												[
													596,
													460
												],
												[
													596,
													390
												],
												[
													687,
													390
												],
												[
													687,
													550
												],
												[
													517,
													550
												]
											]
										},
										{
											"area": "W02DM4",
											"points": [
												[
													231,
													175
												],
												[
													198,
													175
												],
												[
													198,
													186
												],
												[
													165,
													186
												],
												[
													165,
													199
												],
												[
													109,
													199
												],
												[
													109,
													296
												],
												[
													205,
													296
												],
												[
													205,
													268
												],
												[
													231,
													268
												]
											]
										},
										{
											"area": "W02DM5",
											"points": [
												[
													413,
													675
												],
												[
													413,
													566
												],
												[
													295,
													566
												],
												[
													295,
													675
												]
											]
										},
										{
											"area": "W02DM6",
											"points": [
												[
													218,
													702
												],
												[
													218,
													799
												],
												[
													319,
													799
												],
												[
													319,
													702
												]
											]
										}
									],
									"imageType": "map"
```

File: collection/Kobold Press; Warlock 02 Dread Magic.json (hunk 1 above)
Comment (Spappz) [reply collection/Kobold Press; Warlock 02 Dread Magic.json:158]: oh lmao whoops

**Review by TheGiddyLimit — APPROVED** [review]

**Comment by Spappz** [comment]
nooooooo 10 minutes too soon 😩

i'll make another PR

**Comment by revilowaldow** [comment]
> i'll make another PR

If it's just for the tag I'll chuck it in my open draft one dw :kobPet:

**Comment by Spappz** [comment]
nah it's for a bit more _but_ i do have write privileges to your fork so i'll just commit them there 😏

**Comment by revilowaldow** [comment]
Have I still not removed that? :kekw:

**Pull request merged**